osu.Game/Overlays/BeatmapSetOverlay.cs

```diff
@@ -139,6 +139,20 @@ protected override bool OnClick(InputState state)
             return true;
         }
 
+        public void ShowBeatmap(int beatmapId)

```

Comment:
this should probably be done in a separate pull request (and should have the user arrive to the correct difficulty's screen, rather than ignoring that). implementing methods like this without a TODO or an associated issue is carelessly creating bugs due to lacking implementation.

Reply:
Sorry. Got a bit too excited there and just put it in carelessly.